stacks_queues/stack.py

```diff
@@ -12,27 +13,33 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_last(element)        
 
     def pop(self):
         """ Removes an element from the top
             Of the Stack
             Raises a StackEmptyException if
             The Stack is empty.
-            returns None
         """
-        pass
+        if self.store is None:
```

Comment:
`store` is initialized to a LinkedList instance at construction, and should never be `None`. But we could check whether it's empty
```suggestion
        if self.store.empty():
```